### PR TITLE
build: fix build version numbers and use NPM

### PR DESCRIPTION
**Pull request by gnomeontherun**
We need the version numbers to be updated correctly for distribution on NPM. We also don't want to install @clr/icons anymore as it is not used with v5+. We want to have @clr/angular point to @cds/core@^5.0.0, so fixed that in the schematic and package.

I also dropped `yarn` for `npm` as it will make it simpler. 

I tested this locally and it now installs correctly with ng add.

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] If applicable, have a visual design approval

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] clarity.design website / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

**Review by mathisscott — APPROVED** [review]

**Review by coryrylan — APPROVED** [review]

**Comment by gnomeontherun** [comment]
Adding GitHub checks back too.

**Review by Shijir — APPROVED** [review]
Awesome!

File: scripts/website.sh [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+mkdir -p logs
+
+# This downloads a set of key-value env variables, but blocks any requests that aren't from GitHub Actions
+curl -g https://us-central1-clarity-design-system.cloudfunctions.net/actions -o .env
+# dotenv loads up the env variables into the shell, then deploys through Netlify
+
+# Deploy a preview that can be promoted to production when we are ready
+node -r dotenv/config -- ./node_modules/.bin/netlify deploy --json --dir=./dist/website --message="Angualr Branch - Website - $GITHUB_REF@$GITHUB_SHA" --site "clarity.design" > ./logs/clarity.design.netlify.json
+
+echo "Clarity Design Netlify"
```
Comment (bdryanovski) [inline scripts/website.sh]: You don't need the two lines below .. cause we don't build `clarity.design` here so on `Github CI` run there will be no one to create it.

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -0,0 +1,24 @@
+name: Build
```
Comment (bdryanovski) [inline .github/workflows/build.yml]: I already have this change here #6061

File: .github/workflows/dev.yml [hunk 3]
```diff
@@ -0,0 +1,26 @@
+name: Dev
```
Comment (bdryanovski) [inline .github/workflows/dev.yml]: Same here as here #6061

File: .prettierignore [hunk 4]
```diff
@@ -11,3 +11,4 @@ golden/
 .netlify
```
Comment (bdryanovski) [inline .prettierignore:11]: #6061 is mention here as well

File: scripts/website.sh (hunk 1 above)
Comment (gnomeontherun) [reply scripts/website.sh]: This will actually need to be updated when we deploy the website on the subdomain, but that isn't setup yet.

**Review by mathisscott — APPROVED** [review]

**Pull request merged**